testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvMultinodeAllConnectors.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.launcher.env.environment;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.tests.product.launcher.docker.DockerFiles;
+import io.trino.tests.product.launcher.docker.DockerFiles.ResourceProvider;
+import io.trino.tests.product.launcher.env.Environment;
+import io.trino.tests.product.launcher.env.EnvironmentProvider;
+import io.trino.tests.product.launcher.env.common.StandardMultinode;
+import io.trino.tests.product.launcher.env.common.TestsEnvironment;
+
+import javax.inject.Inject;
+
+import java.util.List;
+
+import static io.trino.tests.product.launcher.env.EnvironmentContainers.isPrestoContainer;
+import static io.trino.tests.product.launcher.env.common.Standard.CONTAINER_PRESTO_ETC;
+import static java.util.Objects.requireNonNull;
+import static org.testcontainers.utility.MountableFile.forHostPath;
+
+@TestsEnvironment
+public final class EnvMultinodeAllConnectors
+        extends EnvironmentProvider
+{
+    private final ResourceProvider configDir;
+
+    @Inject
+    public EnvMultinodeAllConnectors(StandardMultinode standardMultinode, DockerFiles dockerFiles)
+    {
+        super(standardMultinode);
+        this.configDir = requireNonNull(dockerFiles, "dockerFiles is null").getDockerFilesHostDirectory("conf/environment/multinode-all/");
+    }
+
+    @Override
+    public void extendEnvironment(Environment.Builder builder)
+    {
+        // blackhole, jmx, tpch are already configured in Standard base env
+        ImmutableList.Builder<String> listBuilder = ImmutableList.builder();
+        List<String> connectors = listBuilder
+                // TODO accumulo needs to connect to ZooKeeper, it won't start otherwise
+                //.add("accumulo")
+                .add("atop")
+                .add("bigquery")
+                .add("cassandra")
+                .add("clickhouse")
+                .add("druid")
```

Comment:
Missing `delta-lake`

Reply:
Done! Rebased on latest master (Trino 375)

Reply:
How can we ensure this stays in sync? We'll forget to add things here

Reply:
Ashhar also asked about that: https://github.com/trinodb/trino/pull/10895#discussion_r830948904